client/components/form/input-select.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint jsx-a11y/no-onchange: 0 */
```

Comment:
Don't disable these rules - we should work on fixing the errors, not ignoring them. (master already has lint errors, we can leave a few more in for now and fix in another PR -- happy to help there too)